### ISSUE TITLE
Home for documentation

### ISSUE THREAD

**Issue by jlndnn** [open-source-siempo/siempo-android-launcher#5]
We need some technical documentation to help contributors who are interested in getting involved with the project. While this would be an ongoing effort, we should set something up to host documentation - I was thinking a readthedocs site or similar

**Comment by jlndnn** [comment]
Thinking for now this could be a contributor.md with some basic local build instructions

**Comment by tyler-hitzeman** [comment]
Agreed that Contributor.md with build instructions would be a great place to start.

**Comment by zed13** [comment]
I've found nice example in glide repository of CONTRIBUTION.md https://github.com/bumptech/glide/blob/master/CONTRIBUTING.md

**Comment by tyler-hitzeman** [comment]
@zed13 @bhanup212 - can either of you create a CONTRIBUTING.md file, so new devs can get started easier?

**Comment by tyler-hitzeman** [comment]
Some other ideas for getting people involved, esp non-devs:

- Setup a form to get more info about people who are interested
    - Only follow-up with those willing to regularly contribute (2hrs/week?)
    - 1-on-1 follow up or encourage them to join the next coworking
- Recommend what tasks they start on
